src/pages/devtools/index.tsx

```diff
@@ -0,0 +1,38 @@
+import ReactDOM from 'react-dom';
+import FormsList from './FormsList';
+
+try {
+  chrome.devtools.panels.create('Dev Tools', 'icon-34.png', 'rc/pages/panel/index.html');
+} catch (e) {
+  console.error(e);
+}
+
+chrome.tabs.query({ active: true, currentWindow: true }, function(tabs) {
```

Comment:
Please make the code more self-explanatory the code it self need to be properly named or commented.
For example.
Instead of this:
```ts
chrome.tabs.query({ active: true, currentWindow: true }, function(tabs) {
  chrome.tabs.sendMessage(tabs[0].id as number, { action: 'readForms' }, function(response) {
    if (response && response.formsData) {
      console.log('Forms Data:', response.formsData);
      ReactDOM.render(<FormsList formsData={response.formsData} />, document.getElementById('root'));
    } else {
      console.log('No forms data received');
    }
  });
});
```
Do this
```ts
import { sendMessage } from './your-message-code';

// This can be exported to some helper file
const chromeTabs = (cb: (tabs) => void) => chrome.tabs.query({ active: true, currentWindow: true }, cb);
const sendMessageToActiveTab = (tabs, cb) => chrome.tabs.sendMessage(tabs[0].id as number, { action: 'readForms' }, cb);
const handleFormDataResponse = (response) => {
  if (response && response.formsData) {
    sendMessage('form-data', response.formsData);
  } else {
    sendMessage('error-form-data', 'No forms data received');
  }
};

chromeTabs((tabs) => sendMessageToActiveTab(tabs, handleFormDataResponse));
```